observer/test/observer.test.js

```diff
@@ -1,19 +1,132 @@
 import assert from 'node:assert'
-import { observeScheduledRewards } from '../lib/observer.js'
+import { beforeEach, describe, it } from 'mocha'
 import { getPgPools } from '@filecoin-station/spark-stats-db'
 import { givenDailyParticipants } from 'spark-evaluate/test/helpers/queries.js'
 
-const getDayAsISOString = d => d.toISOString().split('T')[0]
-const today = () => getDayAsISOString(new Date())
+import { observeTransferEvents, observeScheduledRewards } from '../lib/observer.js'
 
 describe('observer', () => {
-  describe('observeScheduledRewards', () => {
-    let pgPools
+  let pgPools
+  const getDayAsISOString = d => d.toISOString().split('T')[0]
+  const today = () => getDayAsISOString(new Date())
```

Comment:
Was there any particular reason for moving these two helper functions into a `describe` block? Can we move them back to the top where they were before?

Reply:
You're right here as well

Reply:
yes please